jetty-webapp/src/test/java/org/eclipse/jetty/webapp/WebAppContextTest.java

```diff
@@ -254,177 +266,196 @@ public void testNullPath() throws Exception
         server.addConnector(connector);
 
         server.start();
-        try
-        {
-            String response = connector.getResponse("GET http://localhost:8080 HTTP/1.1\r\nHost: localhost:8080\r\nConnection: close\r\n\r\n");
-            assertThat("Response OK", response, containsString("200 OK"));
-        }
-        finally
-        {
-            server.stop();
-        }
+
+        String rawResponse = connector.getResponse("GET http://localhost:8080 HTTP/1.1\r\nHost: localhost:8080\r\nConnection: close\r\n\r\n");
+        HttpTester.Response response = HttpTester.parseResponse(rawResponse);
+        assertThat("Response OK", response.getStatus(), is(HttpStatus.OK_200));
     }
 
     @Test
     public void testNullSessionAndSecurityHandler() throws Exception
     {
-        Server server = new Server(0);
+        Server server = newServer();
+
         HandlerList handlers = new HandlerList();
         ContextHandlerCollection contexts = new ContextHandlerCollection();
         WebAppContext context = new WebAppContext(null, null, null, null, null, new ErrorPageErrorHandler(),
             ServletContextHandler.NO_SESSIONS | ServletContextHandler.NO_SECURITY);
         context.setContextPath("/");
-        context.setBaseResource(Resource.newResource("./src/test/webapp"));
+
+        Path testWebapp = MavenTestingUtils.getProjectDirPath("src/test/webapp");
+        context.setBaseResource(new PathResource(testWebapp));
         server.setHandler(handlers);
         handlers.addHandler(contexts);
         contexts.addHandler(context);
 
         LocalConnector connector = new LocalConnector(server);
         server.addConnector(connector);
 
-        try
-        {
-            server.start();
-            assertTrue(context.isAvailable());
-        }
-        finally
-        {
-            server.stop();
-        }
+        server.start();
+        assertTrue(context.isAvailable());
     }
 
-    class ServletA extends GenericServlet
+    @Test
+    public void ordering() throws Exception
```

Comment:
aren't there a lot more ordering tests somewhere for fragments and things like that?  so perhaps `testMetaInfJarOrder` would be a better name to avoid confusion.

Reply:
Another old test.

It arrived from the community in commit https://github.com/eclipse/jetty.project/commit/1ffbf15fc1caa4182b45f152767ee7e5478c5e58

Which seems to reference `WebAppClassLoaderTest.ordering()` (another name change?)

https://github.com/eclipse/jetty.project/blob/jetty-9.4.31.v20200723/jetty-webapp/src/test/java/org/eclipse/jetty/webapp/WebAppClassLoaderTest.java#L345-L364

Which arrived from commit https://github.com/eclipse/jetty.project/commit/ac61f0e968674728994c5855c5b5ad6c40441ed8

Now that I look at it, these are both attempting to prove JAR ordering in a WebAppContext, uh ... that's not supported.
IMO, both of these `ordering()` tests should be removed.

Reply:
This ordering() test case attempts to prove ClassLoader jar file order.
It's been removed.